Abstract Chg draft-brw-sconepro-rate-policy-discovery.md (#14)
boucadair/draft-xxx-ac-rate-policy-discovery: draft-brw-sconepro-rate-policy-discovery.md

--- abstract

> Traffic exchanged over an access network connection may be subject to rate-limit policies.

Reviewer: ```suggestion
Traffic exchanged over a network attachment may be subject to rate-limit policies.
```

> These policies may be intentional policies (e.g., enforced as part of the activation of the access network connection and typically agreed upon service subscription)

Reviewer: ```suggestion
These policies may be intentional policies (e.g., enforced as part of the activation of the network attachment and typically agreed upon service subscription)
```
